changelogs/unreleased/gh-8108-adjust-xlog_read_ahead.md

```diff
@@ -0,0 +1,4 @@
+## feature/core
+
+* Improved snapshot recovery time by up to 2x on the systems with a hard disk
```

Comment:
Nit: "Improved recovery time" sounds weird to me (I'm not sure if it's incorrect though). I'd say "Reduced recovery time".

Reply:
Done